gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/ApplicationServiceImpl.java

```diff
@@ -233,17 +233,28 @@ public ApplicationEntity findById(final ExecutionContext executionContext, Strin
     }
 
     @Override
-    public Set<ApplicationListItem> findByIds(final ExecutionContext executionContext, Collection<String> applicationIds) {
+    public Set<ApplicationListItem> findByIds(ExecutionContext executionContext, Collection<String> applicationIds) {
```

Comment:
Previously, the findByIds method implicitly searched applications by id and ApplicationStatus.ACTIVE by default. By replacing it with your implementation, you've altered the behaviour for all existing resources or services that currently rely on this method.

Could you please verify whether this method is still being called elsewhere? If so, ensure that the default behaviour hasn't been unintentionally modified. If the change impacts existing functionality, consider using your findByIdsAndStatus method explicitly to preserve the intended behaviour.

Reply:
1. I replaced/renamed findByIds to findByIdsAndStatus and updated all the usages/calls by adding additional parameter: ApplicationStatus.Active
2. Then I added a new method findByIds